src/base/baseAnnotationTool.js

```diff
@@ -1,5 +1,13 @@
 import baseTool from './baseTool.js';
 
+import { getToolState } from './../stateManagement/toolState.js';
+import handleActivator from './../manipulators/handleActivator.js';
+
+import {
+  moveHandleNearImagePoint,
+  moveAnnotationNearClick
+} from './baseAnnotationToolHelpers.js';
```

Comment:
We may need to expose `moveHandleNearImagePoint` and `moveAnnotationNearClick` in a better way for custom tools built outside of this framework; although; the current implementation will probably work fine for the majority of them 🤔

Reply:
We could also expose them in some clever way to be utilised by tools that extend this framwork, maybe even just a reference would suffice to them in e.g. baseAnnotationTool?